docker/buildroot.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -e
+
+docker_image="goranche/raspbian-ua-netinst"
```

Comment:
so build it yourself 👍 you have the dockerfile available

the repository name on docker hub would be changed later on, of course, but for now it's under my username...